deploy/crds/kieapp.crd.yaml

```diff
@@ -1913,6 +1913,32 @@ spec:
                                 type: string
                               type: array
                           type: object
+                        kafkaJbpmEventEmitters:
```

Comment:
There are some configuration endpoints missing that may be handy for users.
As I can see we are missing here property for date format configuration and properties for topics (processes, tasks, cases). See [KafkaEventEmitter](https://github.com/kiegroup/jbpm/blob/master/jbpm-event-emitters/jbpm-event-emitters-kafka/src/main/java/org/jbpm/event/emitters/kafka/KafkaEventEmitter.java)
They are not required to be set.

Reply:
Added date format

Reply:
Still missing topic mapping properties
org.kie.jbpm.event.emitters.kafka.topic.processes
org.kie.jbpm.event.emitters.kafka.topic.cases
org.kie.jbpm.event.emitters.kafka.topic.tasks

Reply:
these are added by the image, seems they are hardcoded, Max, is this the expected behaviour?

Reply:
Yes, are hard coded in the emitter's code ithis call https://github.com/kiegroup/jbpm/blob/master/jbpm-event-emitters/jbpm-event-emitters-kafka/src/main/java/org/jbpm/event/emitters/kafka/KafkaEventEmitter.java#L163 and here https://github.com/kiegroup/jbpm/blob/master/jbpm-event-emitters/jbpm-event-emitters-kafka/src/main/java/org/jbpm/event/emitters/kafka/KafkaEventEmitter.java#L91..L106